packages/tests/web-test-runner.config.js

```diff
@@ -56,16 +56,30 @@ function makeSauceLauncherOnce() {
   if (!sauceLauncher) {
     const user = (process.env.SAUCE_USERNAME || '').trim();
     const key = (process.env.SAUCE_ACCESS_KEY || '').trim();
-    if (!user || !key) {
-      throw new Error(
-        'To test on Sauce, set the SAUCE_USERNAME' +
-          ' and SAUCE_ACCESS_KEY environment variables.'
-      );
+    const tunnelIdentifier = (process.env.TUNNEL_IDENTIFIER || '').trim();
+
+    if (!user || !key || !tunnelIdentifier) {
+      throw new Error(`
+To test on Sauce, set the following env variables
+- SAUCE_USERNAME
+- SAUCE_ACCESS_KEY
+- TUNNEL_IDENTIFIER
+    `);
     }
-    sauceLauncher = createSauceLabsLauncher({
-      user,
-      key,
-    });
+    sauceLauncher = createSauceLabsLauncher(
+      {
+        user,
+        key,
+      },
+      {
+        build: tunnelIdentifier,
+      },
+      {
+        tunnelIdentifier,
+        sharedTunnel: true,
```

Comment:
Could you add brief comments explaining `sharedTunnel` and `noRemoveCollidingTunnels` and why we want to set them here?

Reply:
For sure!